ethereum/core/src/test/java/org/hyperledger/besu/ethereum/mainnet/MainnetTransactionValidatorTest.java

```diff
@@ -300,13 +306,60 @@ public void shouldRejectTransactionWithMaxPriorityFeeGreaterThanMaxFee() {
             .signAndBuild(new SECP256K1().generateKeyPair());
 
     final ValidationResult<TransactionInvalidReason> validationResult =
-        validator.validate(transaction, Optional.of(Wei.ONE), transactionValidationParams);
+        validator.validate(
+            transaction, Optional.of(Wei.ONE), Optional.empty(), transactionValidationParams);
     assertThat(validationResult)
         .isEqualTo(ValidationResult.invalid(MAX_PRIORITY_FEE_PER_GAS_EXCEEDS_MAX_FEE_PER_GAS));
     assertThat(validationResult.getErrorMessage())
         .isEqualTo("max priority fee per gas cannot be greater than max fee per gas");
   }
 
+  @Test
```

Comment:
Is there already a test that check the opposite, when a tx is valid?

Reply:
yes, `shouldAcceptTransactionWithAtLeastOneBlob` covers this with a tx that overpays.